### ISSUE TITLE
hackerspace open/close sign does not load properly

### ISSUE THREAD

**Issue by maxbachmann** [ToolboxBodensee/toolbox-webseite#466]
the hackerspace open/close sign on the the website needs a couple seconds until it appears. It should appear instantly (it did a couple months ago)

**Comment by DO1JLR** [comment]
The reason is, that is now a XHR Script that is parsing the JSON from the SpaceAPI manually.

The CSS is currently not used.

Please talk to @re4jh about this!

**Comment by maxbachmann** [comment]
Yeah thats why I opened an issue, since it is not working properly the way it is currently done ;)

**Comment by DO1JLR** [comment]
Wir könnten das mit dem XHR rückgängig machen... Aber das wäre schon schade für den Aufwand den @re4jh da betrieben hat.

@re4jh hast du hier einen Vorschlag?

**Comment by re4jh** [comment]
da ist ein 5s versatz drin, weil das alle 5 Sekunden aktualisiert. einmalig auf page-load (also das event, wenn die seite initial da ist) vorverlegen sollte helfen.
ich kümmere mich drum.

**Comment by DO1JLR** [comment]
Ich fände CSS als Fallback trotzdem cool!

**Comment by re4jh** [comment]
sollte das nicht in https://github.com/chaos-bodensee/lektor-theme-landed platziert sein, also der issue?

**Comment by DO1JLR** [comment]
würde auch sinn machen...

**Comment by re4jh** [comment]
Dafür gibts jetzt [PR#4 ebenda](https://github.com/chaos-bodensee/lektor-theme-landed/pull/4).

**Comment by re4jh** [comment]
Zu deinem Wunsch nach CSS-Fallback:
https://toolbox-bodensee.de/css/default-spacestatus.css ist dein Freund.

Da müssen ja nicht beide Kandidaten ein display:none bekommen. Der Sichtbare bekommt ein display:block, der Unsichtbare sein display:none ... Fertsch.

War das nichtmal irgendwo verbaut, dass irgendein Prozess diese Datei schreibt, oder gab's da ne ganz andere CSS für? Mein Hirnsieb erinnert sich, dass da CSS-Dateizeuch von extern überschrieben wurde.

**Comment by DO1JLR** [comment]
https://toolbox-bodensee.de/css/spacestatus.css

**Comment by re4jh** [comment]
Uff. Da sind ja Syntaxfehler drin. Kurz und Knapp. Die Raute markiert IDs, der Punkt Klassen. Ohne eins von Beidem müsste "space-opened" ein Tag sein. Ich halte das aber für eine ID. Müsste man also mit ner Raute prefixen. Macht da bitte jemand mal #space-opened draus. Dann hatt das Ding die nötigen Ersatzmaterialqualitäten. Merci.
